spyder/plugins/updatemanager/workers.py

```diff
@@ -294,6 +300,10 @@ def start(self):
         except HTTPError as err:
             error_msg = HTTP_ERROR_MSG.format(status_code=page.status_code)
             logger.warning(err, exc_info=err)
+        except OSError as err:
+            error_msg = OS_ERROR_MSG
```

Comment:
```suggestion
            error_msg = OS_ERROR_MSG.format(error=err)
```

To go with my suggestion above.